### ISSUE TITLE
Early Error rules need to be referenced explicitly in PerformEval

### ISSUE THREAD

**Issue by jorendorff** [tc39/proposal-class-fields#234]
The proposal contains [new early error rules](https://tc39.github.io/proposal-class-fields/#sec-performeval-rules-in-initializer) for direct eval inside initializers.

Steps 4-6 of PerformEval should be updated to specify when these rules are actually applied, as those steps already do for `new.target` and `super`.

**Comment by littledan** [comment]
Good point. PRs welcome, if anyone is interested in helping with this specification, cc @xanlpz @caitp
